Console/Command/GraphShell.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-App::uses('AppShell', 'Console\Command');
+App::uses('AppShell', 'Console/Command');
```

Comment:
@kimegede maybe it's better to use DS or DIRECTORY_SEPARATOR constant here, to make sure it works both on Windows and Linux/Mac.